source/_integrations/aidot.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Aidot
+description: Instructions on integrating Aiot home security with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.6
+ha_iot_class: local_polling
+ha_config_flow: true
+ha_codeowners:
+  - '@s1eedz'
+ha_domain: Aidot
+ha_homekit: true
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+The [Aidot](https://www.aidot.com/) {% term integration %} allows you to control the Wi-Fi lights with Home Assistant.
+
+{% include integrations/config_flow.md %}
```

Comment:
the instructions are already in the Prerequisites section, so they can be removed here to avoid duplicates